_posts/2019-08-12-how-to-configure-production-grade-aws-account-structure.adoc

```diff
@@ -803,6 +803,81 @@ the central place where you manage billing. You create this initial account manu
 . You will be asked to enter an email address and password to use as the credentials for the root user of this root
   account.
 
+=== Configure your Terraform remote state S3 bucket and DynamoDB lockfile
+Now we need to configure the backend for your root account to store your terraform state in an s3 bucket and the dynamoDB table for the lockfile.
+Create a terragrunt.hcl file in infrastructure-live/root.
```

Comment:
Hm, you need such a file in the root folder of each account. I omit it because one of the pre-requisites for this guide is that you read the [How to use the Gruntwork Infrastructure as Code Library
](https://gruntwork.io/guides/foundations/how-to-use-gruntwork-infrastructure-as-code-library/#modules), which includes [instructions on how to configure Terragrunt](https://gruntwork.io/guides/foundations/how-to-use-gruntwork-infrastructure-as-code-library/#using_terraform_modules) (including this root `terragrunt.hcl` file). Perhaps instead of making a copy of all of that, we cross-link to those docs? That is, if you search for the text "Pull in the backend settings from a root terragrunt.hcl file" in this guide, that would be a good place to cross-link to the [instructions on how to configure Terragrunt](https://gruntwork.io/guides/foundations/how-to-use-gruntwork-infrastructure-as-code-library/#using_terraform_modules).

Reply:
Thank you for the feedback.  I missed that section in the first guide. I will incorporate your suggestions.

Reply:
Thanks! Let me know when that's ready and I'll get this merged and deployed.

Reply:
@brikis98  - After reviewing the  [How to use the Gruntwork Infrastructure as Code Library](https://gruntwork.io/guides/foundations/how-to-use-gruntwork-infrastructure-as-code-library/).  I think the users need a fundamental understanding of that document before they attempt these installs.  Your initial version is probably the correct pattern to follow.